lib/puppet/functions/vault_sorted_json.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
+
+# This function takes data, outputs making sure the hash keys are sorted
+#
+# *Examples:*
+#
+#     sorted_json({'key'=>'value'})
+#
+# Would return: {'key':'value'}
+Puppet::Functions.create_function(:vault_sorted_json) do
```

Comment:
Does this need to be `create_function(:'vault::vault_sorted_json')` per https://www.puppet.com/docs/puppet/7/functions_refactor_legacy.html#functions_refactor_legacy-namespacing ?

Reply:
It works without namespacing, but we should probably move it to be namespaced like that for the benefit of future generations, good suggestion.